py/selenium/webdriver/chromium/service.py

```diff
@@ -56,5 +58,15 @@ def __init__(
             **kwargs,
         )
 
+    @property
+    def service_args(self):
```

Comment:
be nice to add typing to things we are adding, feel free to ignore tho as theres plenty of general typing work to do

Reply:
@symonk thanks for your review comments. I have updated the code.